app.yaml

```diff
@@ -15,34 +15,36 @@ inbound_services:
 
 handlers:
 - url: /favicon.ico
-  static_files: static/images/favicon.ico
-  upload: static/images/favicon.ico
+  static_files: assets/common/favicon.ico
+  upload: assets/common/favicon.ico
   secure: always
   http_headers:
     Cache-Control: 'public, max-age=2592000'
     Vary: Accept-Encoding
-- url: /images
-  static_dir: static/images
+- url: /robots.txt
+  static_files: assets/common/robots.txt
+  upload: assets/common/robots.txt
   secure: always
   http_headers:
-    Cache-Control: 'public, max-age=600'
-- url: /static/pages
-  static_dir: static/pages
+    Cache-Control: 'public, max-age=2592000'
+    Vary: Accept-Encoding
+- url: /build
+  static_dir: build
   secure: always
   http_headers:
-    Cache-Control: 'public, max-age=600'
-- url: /robots.txt
-  static_files: static/pages/robots.txt
-  upload: static/pages/robots.txt
+    Cache-Control: 'public, max-age=2592000'
+- url: /assets/common
+  static_dir: assets/common
   secure: always
   http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
-- url: /scripts
-  static_dir: static/scripts
+    # WARNING TO DEVELOPERS: Files in this folder may be stale for
+    # up to an hour after a code release.
+    Cache-Control: 'public, max-age=3600'
+- url: /assets
```

Comment:
Why are we caching this directory? I thought all the cached stuff goes in /build.